nautobot_floor_plan/svg.py

```diff
@@ -174,117 +180,180 @@ def _draw_grid(self, drawing):
                     )
                 )
 
+    # draw edit and delete functionality to Status and Rackgroup tiles
+    def _draw_defined_status_tiles(self, drawing, tile):
+        """Render a tile based on its Status."""
```

Comment:
Could you update the docstring to clarify what this method is doing and how it differs from `_draw_status_tiles`?